src/hosted.js

```diff
@@ -131,12 +133,15 @@
     }
 
     // Support loading from src/ or dist/ to make local dev easier
-    require(["bramble/client/main"], function(Bramble) {
+    if(window.location.pathname === "/dist/hosted.html") {
```

Comment:
this won't work on staging/production because of S3 object prefixes.
You would probably need to regex this instead (or to be safer, have a `meta` tag in the html indicating that it is a hosted version and query that)